macros/edr/dbt_artifacts/upload_dbt_invocation.sql

```diff
@@ -41,6 +51,46 @@
     {% do return(config.project_name) %}
 {% endmacro %}
 
+{% macro get_env() %}
```

Comment:
Isn't a function that returns functionB(var) is a bit redundant?
I can see why making those macros is good when we want to use them in other cases as well, which in that case I think they should be under env_utils or something like that 🙂

Reply:
Actually yeah, I agree.
These functions were longer beforehand but now that it's only a single expression,
makes more sense. I'll change.